apps/vc-api/src/vc-api/vc-api.controller.ts

```diff
@@ -133,6 +135,11 @@ export class VcApiController {
     @Param('exchangeId') exchangeId: string,
     @Param('transactionId') transactionId: string
   ) {
-    return new NotImplementedException();
+    const queryResult = await this.exchangeService.getExchangeTransaction(transactionId);
+    const response: GetTransactionDto = {
+      errors: queryResult.errors,
+      transaction: queryResult.transaction ? TransactionDto.toDto(queryResult.transaction) : undefined
```

Comment:
Is there a better way/place to do this entity to DTO conversion? This isn't that elegant 😓

Reply:
I can't think of better way rn. Maybe just move it after `queryResult` like `const transactionDto = queryResult.transaction ? TransactionDto.toDto(queryResult.transaction) : undefined`